services/http/api/lib/interface/index.js

```diff
@@ -33,4 +33,13 @@ module.exports = {
             {...m, time, day, month, year })
         return user.records.length > 0
     },
+
+    createSurvey: (s) => async (db) => {
+        const date = new Date(s.date * 1000)
+        const year = date.getFullYear()
+        const month = date.getMonth()
+        const day = date.getDate()
+        const time = 3600*date.getHours() + 60*date.getMinutes() + date.getSeconds()
```

Comment:
Can we abstract this to a util function and use it in the function above?

Reply:
Sounds like a good idea. 

```.js
function dateTransformer(input) {
    const date = new Date(input * 1000);
    const year = date.getFullYear();
    const month = date.getMonth();
    const day = date.getDate();
    const time = 3600 * date.getHours() + 60 * date.getMinutes() + date.getSeconds();
    
    return {
        year, 
        month,
        day, 
        time
    }
}

```

This was added!